src/Uplink/Admin/Assets.php

```diff
@@ -0,0 +1,47 @@
+<?php declare( strict_types=1 );
+
+namespace StellarWP\Uplink\Admin;
+
+use StellarWP\Uplink\Config;
+
+class Assets {
```

Comment:
Minor, but avoid using plurals for what is a single object.

```suggestion
class Asset_Manager {
```